src/borg/compress.pyx

```diff
@@ -346,6 +382,16 @@ class CompressionSpec:
             else:
                 raise ValueError
             self.level = level
+        elif self.name in ('zstd', ):
+            if count < 2:
+                level = 5  # default compression level for zstd
```

Comment:
taken "as is", but it seems like the default level in python-zstandard is rather 3.

so, take 3 or use a different level better for typical borg usecase?